### PR TITLE
Fix issue #122

### PR DESCRIPTION
**Pull request by floriansegginger**
Return the jQuery element in wysiwyg function and expose cleanHtml as a jQuery function.

### CODE REVIEW

**Comment by codewithtyler** [comment]
Hi @floriansegginger your last commit changes the bower package name for your PR. When your PR gets merged this will affect the repository. Can you set it back to the previous name?

**Comment by floriansegginger** [comment]
I’m sorry, I got a bit confused trying to make my own bower package .. this is my first time doing this :)

I don’t really care too much about the credit. Since this is a small change maybe you can make the commit yourself and close the pull request?

-- 
	
Florian Segginger
Co-founder, Fullrange Interactive Sàrl
+41 79 375 65 66
florian@fullrange.ch <mailto:florian@fullrange.ch> | f@floriansegginger.ch <mailto:f@floriansegginger.ch>
Facebook <http://www.facebook.com/fullrangeinteractive> | www.fullrange.ch <http://www.fullrange.ch/>

**Review by steveathon — CHANGES_REQUESTED** [review]
Hi,

I think in general your change for adding the cleanHTML function seems sane, but I can't approve it with the updated bower. 

Can we strip out the change and edit it in?

File: bower.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "bootstrap-wysiwyg",
+  "name": "floriansegginger-bootstrap-wysiwyg",
```
Comment (steveathon) [inline bower.json:2]: This can't merge into the main package.

File: bower.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
   "description": "A tiny Bootstrap and jQuery based WYSIWYG rich text editor based on the browser function execCommand.",
-  "version": "1.0.4",
+  "version": "2.1.0",
```
Comment (steveathon) [inline bower.json:4]: This can't merge into the main package.

File: bower.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
     "wysiwyg",
     "editor"
   ],
-  "homepage": "https://github.com/steveathon/bootstrap-wysiwyg",
+  "homepage": "https://github.com/floriansegginger/bootstrap-wysiwyg",
```
Comment (steveathon) [inline bower.json:14]: This can't merge into the main package.

**Comment by mhagnumdw** [comment]
When will the next version come out? I use version 2.0.1 and I can not access cleanHtml. Tks!

File: bower.json (hunk 1 above)
Comment (steveathon) [reply bower.json:2]: Can you update this so we can merge?

File: bower.json (hunk 2 above)
Comment (steveathon) [reply bower.json:4]: Also this.

File: bower.json (hunk 3 above)
Comment (steveathon) [reply bower.json:14]: Also this.

**Review by steveathon — COMMENTED** [review]
Overall I'm happy to merge this I'm just waiting for the reviewed changes. Either I'll wait for this update, or I'll cherry pick the update.

**Comment by steveathon** [comment]
Only just saw your other comment, apologies @floriansegginger - I've created the branch /cleanHTML and I'll be updating it shortly.

Thanks for your contribution!

**Pull request closed without merging**